src/Microsoft.IdentityModel.Logging/TextWriterEventListener.cs

```diff
@@ -52,35 +54,34 @@ public TextWriterEventListener()
             }
             catch (Exception ex)
             {
-                LogHelper.Throw(string.Format(CultureInfo.InvariantCulture, LogMessages.MIML11001, ex.Message), ex.GetType(), EventLevel.Error, ex.InnerException);
+                LogHelper.LogException<InvalidOperationException>(ex, LogMessages.MIML11001);
+                throw ex;
```

Comment:
LogHelper.LogException doesn't throw, and I wasn't convinced that InvalidOperationException would give a good message to the user. It was just a call that throwing the existing was probably as good as doing anything else.